### DISCUSSION TITLE
ESPHOME - How to enable https?

### DISCUSSION THREAD

**Discussion by Diondk** [tteck/Proxmox#3570] [Help Needed] [answered]
Hi, 
how can i enable https in the esphome configuration? 
When i want to add the esphome webpage as dashboard to my home assistant it gives an error that it cannot be added because its an http website.

Any help is appreciated.!

Kr, 
Dion

**Comment by Mirabis** [comment, marked as answer]
You should add a reverse proxy in front of it - and most likely your other services as well. The tteck collection already includes a few options for you there:

- Caddy
- BunkerWeb
- Nginx Proxy Manager
- Traefik

**Reply by Diondk** [reply]
Ok hoped it was able in the container, but then i will add a proxy. Thanks!
